<!-- -->
<launch>

    <node name="lidar_mot_node" type="lidar_mot_node.py" pkg="lrm_lidar_object_tracking" >

	<param name="in_detected_objects" value= "/carina/perception/dataset/obstacles_marker_array" />
	<param name="out_detected_objects_markers" value= "/carina/perception/mot/lidar/markers"/>
	<param name="out_detected_objects_obstacles" value= "/carina/perception/mot/lidar/obstacles"/>    
	<param name="expected_frame_rate" value= "10.0" /> <!-- Hz-->
    </node>
</launch>
